configs.py

```diff
@@ -0,0 +1,45 @@
+# HOST = 'https://dev.api.it-sl.ru'
+
+
+HOST = "http://localhost:8083"
+# HOST = 'https://iced-latte.uk/backend'
+HOST_DB = "192.168.1.66"
```

Comment:
This seem to be your inner ip, that will be different for others. Is it possible to use `localhost` there?

Reply:
Resolved